File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int[][] answer = new int[arr1.length][arr1[0].length];
        for(int x = 0; x<arr1.length; x++)
        {
            for(int y = 0; y<arr1[x].length; y++)
                {
                    answer[x][y] = arr1[x][y] + arr2[x][y];
                }

        }
        return answer;
    }
}

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.js
function solution(arr1, arr2) {
    var answer = arr1;
    
    for(let x = 0; x<arr1.length; x++)
    {
        for(let y = 0; y<arr1[x].length; y++)
        {
            answer[x][y] = arr1[x][y] + arr2[x][y];
        }

    }
    
    return answer;
}